explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts

```diff
@@ -1009,6 +1009,20 @@ export function getProjectBreadcrumbs(
   return breadcrumbs;
 }
 
+/**
```

Comment:
File format is unique (it is part of the object `fileTypeSummaries`) and so we will add the function in the `viewModelBuilders.ts` file for re-use.